pom.xml

```diff
@@ -445,6 +452,7 @@
                 org.apache.avro.mapreduce;
                 parquet.avro.*;
                 parquet.hadoop.*;
+                org.apache.spark.streaming.*;
```

Comment:
It doesn't seem like this would be needed. Were you getting errors without it?

Reply:
Yes. Without these I get class not found exception for org.apache.spark.streaming.pubsub.PubsubReceiver. 
I can be try being more specific to pubsub package level